contrib/auto-completion/snippets/emacs-lisp-mode/.yas-parents

```diff
@@ -0,0 +1 @@
+prog-mode
```

Comment:
I guess that it inherits stuff from prog-mode.. But I don't really use anything from it. I copied this from ansible-snippets. Didn't know how yasnippets worked then.